.dockerignore

```diff
@@ -2,9 +2,10 @@
 config/Kv*
 
 #Ignore build directories
-workloads/build*
-tcs/core/common/build*
-tcs/core/tcs_trusted_worker_manager/enclave/build*
-tcs/core/common/python/build*
-common/build*
+tc/sgx/trusted_worker_manager/enclave/build*
+common/sgx_workload/build*
+tc/sgx/common/build*
+examples/enclave_manager/build*
+examples/common/python/build*
+examples/apps/build*
```

Comment:
Will the build directories still created in host machine or are build artifacts created only in docker image and visible only to the container ? If build directories are visible only to container then can we remove these lines ?

Reply:
Build artifacts will not be created on the host machine and are only created in the docker image.
I will remove build directories from dockerignore file.